core/server/src/main/java/alluxio/worker/file/UnderFileSystemManager.java

```diff
@@ -152,6 +197,22 @@ public void cancelFile(long tempUfsFileId) throws FileDoesNotExistException, IOE
   }
 
   /**
+   * Closes an open input stream associated with the given temporary ufs file id. The temporary
+   * ufs file id will be invalid after this is called.
+   *
+   * @param tempUfsFileId the temporary ufs file id
+   * @throws FileDoesNotExistException if the worker is not reading the specified file
+   * @throws IOException if an error occurs when operating on the under file system
+   */
+  public void closeFile(long tempUfsFileId) throws FileDoesNotExistException, IOException {
```

Comment:
The overall naming was to be similar to the `open`, `close` pair and the `create`, `complete` or `cancel` pair we use on the client side. As discussed offline, I will leave a TODO here.